File: Python/7py 2/generate_integers.py
"""
|--------------------------------------------------------------------------
| Series of integers from 0 to n
|--------------------------------------------------------------------------
|
"""

def generate_integers(n):
   res = []
   for i in range(n+1):
      res.append(i)
   return res


def generateIntegers(n):
   return list(range(n + 1))


def generateIntegers(n):
   return [*range(n+1)]


def generateIntegers(n):
   return [a for a in range(n+1)]


print(generate_integers(3), [0, 1, 2, 3])

File: JavaScript/7js 2/generateIntegers.js
/*
|--------------------------------------------------------------------------
| Series of integers from 0 to n
|--------------------------------------------------------------------------
|
*/

function generateIntegers(n) {
   let res = [];
   for (let i = 0; i <= n; i++) {
      res.push(i)
   }
   return res;
}


const generateIntegers = (n) => [...Array(n + 1).keys()]


function generateIntegers(n) {
   return [...Array(n + 1)].map((_, i) => i)
}


function generateIntegers(n) {
   return Array.from({
      length: n + 1
   }, (_, i) => i)
}


console.log(generateIntegers(3), [0, 1, 2, 3]);